### ISSUE TITLE
macOS 13 - Ventura

### ISSUE THREAD

**Issue by BSPlatt** [zearp/OptiHack#80]
Any plans/progress on installing a SMBIOS compatible with macOS Ventura?

Thanks for all your work on this project!

**Comment by zearp** [comment]
Apple has dropped native Haswell support in Ventura and newer. That means things such as frequencies, voltages and other low level things are no longer natively supported by the kerne land could lead to all kinds of issues. Things may also just work fine but as long as Monterey receives security updates I'm not going to worry about it.

When the time comes those security updates stop, or for those feeling adventurous it should be possible to use OCLP but that also has some downsides mainly in regards to system security. I may at some point do some tests and see if an SMBIOS not made for our platform is reliable but it takes some times and effort. Things like monitoring voltages, frequencies and other things.

I recall from when testing 4k stuff that changing the SMBIOS would result in lower or higher base clocks on the iGPU. These kind of things need to be tested properly before just using it, unless you don't mind it being unstable or potentially running hardware at too low, or worse too high clock speeds.

Hopefully using OCLP patches will help prevent such issues but the only way to find out is to test it and at the Monet I don't have the will or need for it. My machines are fine on Monterey and some still Catalina. I sue my NUCs for the things I really need the latest macOS for. 

tl;dr Sonoma/Ventura will likely run using an SMBIOS not made for our platform and the current EFI but stability and security may be impacted.
